.gitignore

```diff
@@ -11,11 +11,13 @@ config.log
 
 # Built by sconsUtils
 version.py
-bin/
 
 # Pytest
 tests/.tests
 pytest_session.txt
 .cache/
 .pytest_cache
 .coverage
+
+.vscode
+ospl-*
```

Comment:
I see that you pushed an ospl-info.log file in the bin directory. Please delete that file.